### PR TITLE
Update PBC to most recent upstream version

### PR DESCRIPTION
**Pull request by a-drees**

### CODE REVIEW

**Comment by a-drees** [comment]
I think it might be worthwhile to revisit how we build our internal prebid cache docker image, and maybe use a repo that isn't a fork.

As it stands right now, the only changes we've made (which of course have conflicts) are to configuration, and the dockerfile.

We are currently not using the graphite changes (which are causing a large amount of the merge conflicts).

I would propose we nuke this repo, saving our configuration (and possibly a couple of choice patches if need-be), and move it to a new repository, where we can also merge in the cloudformation template as well, so we can keep the dockerfile, configuration and cloudformation together.

**Comment by calebgerlachsortable** [comment]
> I think it might be worthwhile to revisit how we build our internal prebid cache docker image, and maybe use a repo that isn't a fork.
> 
> As it stands right now, the only changes we've made (which of course have conflicts) are to configuration, and the dockerfile.
> 
> We are currently not using the graphite changes (which are causing a large amount of the merge conflicts).
> 
> I would propose we nuke this repo, saving our configuration (and possibly a couple of choice patches if need-be), and move it to a new repository, where we can also merge in the cloudformation template as well, so we can keep the dockerfile, configuration and cloudformation together.

I agree that this is worth a revisit. We don't want to spend time maintaining our own fork and we certainly do not want it to go stale and have all the associated problems with that.

So any changes that we make could then be evaluated under two different criteria: 1) Does it provide value for the community as a whole? and 2) Is is necessary? 

For 1, if we have a change that will be valuable to the community as a whole, we should contribute that change back to the open source project. Since Prebid Cache is heavily related to what we do here at Sortable but not core to our value proposition, I would propose that we (as a company) become active in the project by contributing changes back and reaping the benefits of others making changes. I know that I made a trivial change in this repo, (that I think is still unmerged), that would have value for the project as a whole.

For 2, things like the Dockerfile changes or config file changes do not really provide us value. We could use standard Prebid-Cache and distribute a Docker Compose file for local dev (and Cloud Formation config for cloud) that starts the stock Dockerfile with the configuration we require (the project uses Viper for config, which means all stock configuration can be easily overwritten without changing any files).

So in summary, I agree with Alan that we should nuke this repo and solve any problems by contributing back to the project and using the stock container with our own config

**Comment by a-drees** [comment]
If we don't need the configuration file (read: it can be overridden with env variables alone), then we can move the whole thing to a docker-compose file for development, a cfn stack for aws infra and change just the circleci job to trigger updates of the image/configuration.

**Comment by a-drees** [comment]
Going to close this, as it's gotten pretty stale and we have tickets cut for implementing a lot of what was discussed here:

https://sortable.atlassian.net/browse/SOR-4707
